pkg/sfu/buffer/rtpstats_receiver.go

```diff
@@ -359,6 +359,11 @@ func (r *RTPStatsReceiver) GetRtcpReceptionReport(ssrc uint32, proxyFracLost uin
 		fracLost = proxyFracLost
 	}
 
+	totalLost := r.packetsLost
+	if totalLost > 0xffffff { // 24-bits max
+		totalLost = 0xffffff
+	}
```

Comment:
Noticed this when working on pion/interceptor change to fix out-of-order wrap around handling. Adding it for us too.